spec/features/user_views_list_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+feature 'User views list' do
+  scenario "and see's the list's books" do
+    list = create(:list_with_books, books_count: 2)
+
+    visit lists_path
```

Comment:
@samnang when you do feature specs, do you usually start on a specific page or do you hit the `root_path` and navigate to the page you're testing?

I like to start with the root_path, but it's probably not always necessary and adds an extra page that makes the test a bit slower. up to you guys!